controllers/spacebindingcleanup/spacebinding_cleanup_controller_test.go

```diff
@@ -34,12 +43,15 @@ func TestDeleteSpaceBinding(t *testing.T) {
 
 	t.Run("lara-redhat SpaceBinding removed when redhat space is missing", func(t *testing.T) {
 
-		reconciler, request, fakeClient := prepareReconciler(t, sbLaraRedhatAdmin, sbJoeRedhatView, sbLaraIbmEdit, laraMur, joeMur, ibmSpace)
+		fakeClient := test.NewFakeClient(t, sbLaraRedhatAdmin, sbJoeRedhatView, sbLaraIbmEdit, laraMur, joeMur, ibmSpace)
+		reconciler := prepareReconciler(t, fakeClient)
 
 		// when
-		_, err := reconciler.Reconcile(context.TODO(), request)
+		res, err := reconciler.Reconcile(context.TODO(), requestFor(sbLaraRedhatAdmin))
 
 		// then
+		require.False(t, res.Requeue)
```

Comment:
You don't need that when you check the `RequeueAfter`

Reply:
what happens if the controller returns:
```
	return ctrl.Result{
		Requeue:      True,
		RequeueAfter: 0,
	}
```
I was thinking that we should cover this case. But I can remove it.

Reply:
it would immediately schedule another reconcile (with no delay).
but I would skip this check as we don't set anything and we keep the value False by default. Let's do the check only when we set True for some use-cases.

Reply:
or, if you would like to keep it there, you could do:
```go
var (
	noRequeue = reconcile.Result{
		RequeueAfter: 0 * time.Second,
	}
	requeue = reconcile.Result{
		RequeueAfter: 10 * time.Second,
	}
)


...
...
// for the test when no requeue is expected
require.Equal(t, res, noRequeue)

...
...
// for the test when requeue is expected
require.Equal(t, res, requeue)
```

in this way you would easily check both values

Reply:
Hmm yeah, this second option makes sense, thanks! But let's keep it simple and remove it if it doesn't provide any additional value in kind of test coverage.

Reply:
Done, plz check https://github.com/codeready-toolchain/host-operator/pull/857/commits/390a54d71416dbd910a56b5cce82757d1b514cc8